### ISSUE TITLE
Only 95 textures can be used from a single .utx package

### ISSUE THREAD

**Issue by Calypto** [EliotVU/UT2004-MutEmoticons#1]
Hello, if I attempt to use more than 95 textures/emotes from a single .utx package, emotes #96 and up will not show up in game and will give this error in the log (for every emote/texture >95):
`Error: Smileys::ImportText failed in: (Event="coldface",Icon=Texture'OmniEmoticons.1f976')`

I am using the standard MutEmoticons.u file as provided. I can provide the .ini and .utx if needed.

**Comment by EliotVU** [comment]
That's unfortunate, this might be because of an engine limitation that caps any ini-line to a maximum of 1024 characters. Can I see the complete ini source?

**Comment by EliotVU** [comment]
If so, a possible work around could be done by configuring your smileys right into the defaultproperties block of the source code and re-compile the .u file yourself.

**Comment by Calypto** [comment]
I think I figured it out. When importing many textures into the editor and using "Ok All", it appears to stop after #95. I looked in the texture browser under the package and those emotes were indeed missing. This time I selected the ones that were missing, and they successfully imported. I saved the package and tested in game, and it worked.

So for future reference, the import function in the editor can only handle 95 textures at a time. Thank you for your response.
